packages/gollm/gollm_openai/tool_utils.py

```diff
@@ -59,6 +60,47 @@ def get_image_format_string(image_format: str) -> str:
     return format_strings.get(image_format.lower())
```

Comment:
This `return dict.get()` look like it will return a string based on `format_strings` format.
Why change the return type to be a `dict`?

Reply:
good catch. I didn't want to change this one